### PR TITLE
fix/profile-dont-wait-for-avatar

### PR DESCRIPTION
**Pull request by sean-brydon**
Fixes: https://threads.com/thread/34544006411/34544006409?s=kFN7CaE1EA6bZRgApH3KNY

Loading avatar was blocking - instead we just show the fallback avatar until it loaded instead of blocking the whole profile form

### CODE REVIEW

**Comment by sean-brydon** [comment]
Current dependencies on/for this PR:
* main
  * **PR #12080** <a href="https://app.graphite.dev/github/pr/calcom/cal.com/12080" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/calcom/cal.com/12080?utm_source=stack-comment).

File: apps/web/pages/settings/my-account/profile.tsx [hunk 1]
```diff
@@ -226,10 +226,16 @@ const ProfileView = () => {
     [ErrorCode.ThirdPartyIdentityProviderEnabled]: t("account_created_with_identity_provider"),
   };
 
-  if (isLoading || !user || fetchedImgSrc === undefined)
+  if (isLoading || !user) {
+    console.log({
+      user,
+      isLoading,
+    });
```
Comment (sean-brydon) [inline apps/web/pages/settings/my-account/profile.tsx:233]:
```suggestion
```

File: apps/web/pages/settings/my-account/profile.tsx [hunk 2]
```diff
@@ -78,8 +78,8 @@ type FormValues = {
   bio: string;
 };
 
-const checkIfItFallbackImage = (fetchedImgSrc: string) => {
-  return fetchedImgSrc.endsWith(AVATAR_FALLBACK);
+const checkIfItFallbackImage = (fetchedImgSrc?: string) => {
+  return !fetchedImgSrc || fetchedImgSrc.endsWith(AVATAR_FALLBACK);
```
Comment (sean-brydon) [inline apps/web/pages/settings/my-account/profile.tsx:82]: Use fallback if fetchedImgSrc is undefined (loading)

**Review by Udit-takkar — APPROVED** [review]
LGTM. This works great

**Pull request merged**